test/functionalTest/cases/2769_invalid_regular_expression_for_idPattern/invalid_regular_expression_for_idPattern.test

```diff
@@ -0,0 +1,93 @@
+# Copyright 2017 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
It seems this exact case is already covered by cases/2257_crash_with_invalid_subscription_when_matching/crash_with_invalid_subscription_when_matching.test (wihouth step 2 in that case).